src/api/cookie.js

```diff
@@ -0,0 +1,27 @@
+/* eslint no-useless-escape: 0 */
+// @flow
+export function getCookie(name: string): ?string {
+  const matches = document.cookie.match(
+    new RegExp(`(?:^|; )${name.replace(/([\.$?*|{}\(\)\[\]\\\/\+^])/g, '\\$1')}=([^;]*)`),
+  );
+  return matches ? decodeURIComponent(matches[1]) : undefined;
+}
+
+export function setCookie(name: string, value: string, options: Object = {}) {
```

Comment:
I think this library will be much easier than doing it manually:
https://github.com/js-cookie/js-cookie